daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/ProtoTest.scala

```diff
@@ -206,7 +206,7 @@ class ProtoTest extends AnyWordSpec with Matchers with TableDrivenPropertyChecks
     s"daml_lf_1_$version files" should {
 
       val pkg = if (version < 11) "digitalasset" else "daml"
-      val rootDir = s"daml-lf/archive/src/main/protobuf/com/$pkg/daml_lf_1_$version"
+      val rootDir = s"daml-lf/archive/src/snapshot/protobuf/com/$pkg/daml_lf_1_$version"
```

Comment:
maybe `stable` would be clearer? We have snapshot SDK releases and they are something very different so reusing the name might be confusing.

Reply:
19b5cfbc923d46df86c353fa265ebdbdcd2d55b8